spec/foodzone/venues/creator_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative "../../../app/foodzone/venues/creator"
+
+describe Venues::Creator do
+  let(:creator) { described_class }
+  let(:venue_repository) { double :venue_repository }
+
+  describe "::execute" do
+    let(:name) { "The Burguer Shop" }
+    it 'creates a new venue' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.